Tools/SwiftBrowser/SwiftBrowser.xcodeproj/project.pbxproj

```diff
@@ -18,8 +20,23 @@
 		07A58C4A2D0567A100CAB4ED /* SettingsView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 07A58C492D0567A100CAB4ED /* SettingsView.swift */; };
 /* End PBXBuildFile section */
 
+/* Begin PBXCopyFilesBuildPhase section */
+		07A4CEC52D092C0800764F5E /* Embed Frameworks */ = {
+			isa = PBXCopyFilesBuildPhase;
+			buildActionMask = 2147483647;
+			dstPath = "";
+			dstSubfolderSpec = 10;
+			files = (
+				07A4CEC42D092C0800764F5E /* _WebKit_SwiftUI.framework in Embed Frameworks */,
+			);
+			name = "Embed Frameworks";
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXCopyFilesBuildPhase section */
+
```

Comment:
I don't think this is needed—did you intentionally add it, or is it just the default "Embed" phase that Xcode created? We don't intend to ship SwiftBrowser.app as a standalone application, so it does not need to bundle the frameworks it's using (and if it did, we'd likely want to bundle the entire WebKit stack).

Reply:
I don't recall intentionally adding this. Where in Xcode UI would I remove this?

Reply:
SwiftBrowser.xcodeproj > SwiftBrowser > Build Phases. It probably got added when you told it to link against _WebKit_SwiftUI.framework (and presumably _that_ was intentional, for build ordering?)